MonitorAlert/run.ps1

```diff
@@ -0,0 +1,150 @@
+<#
```

Comment:
This is the main file of the function.

It essentially does the following steps:
* grabs inputs
* validates inputs
* generates slack message
* posts slack message